docs/make-a-widget.md

```diff
@@ -138,6 +138,13 @@ Follow these steps for each of the predefined widget types described in this doc
   </value>
 </portlet-preference>
 
+#### Configuring your list of links from a URL
+
+* Rather than hardcode your links into a portlet preference, you can point to a URL and get your links dynamically. 
+1. Omit the "links" entry in the widgetConfig JSON.
+2. In widgetConfig, add the following:
+   "getLinksURL": "true"
```

Comment:
Feasible to make setting the `widgetURL` have this configuration effect, rather than also introducing `getLinksURL`? Configuration that can't be ambiguous is nice -- with the setting overlap, one could set `widgetURL` and have it have no effect because overlooked setting `getLinksURL`.